docs/content/docs/custom-resource/pod-template.md

```diff
@@ -113,10 +112,10 @@ arr1: [{name: a, p2: v2}, {name: c, p2: v2}]
 merged: [{name: a, p1: v1, p2: v2}, {name: c, p1: v1, p2: v2}]
 ```
 
-The operator supports an alternative array merging mechanism that can be enabled by the `kubernetes.operator.pod-template.merge-arrays-by-name` flag.
-When true, instead of the default positional merging, object array elements that have a `name` property defined will be merged by their name and the resulting array will be a union of the two input arrays.
+Operator 支持另一种数组合并机制，可以通过 `kubernetes.operator.pod-template.merge-arrays-by-name` 标志启用。
+当为 true 时，将按名称合并具有 `name` 属性定义的对象数组元素，并且生成的数组将是两个输入数组的并集。
```

Comment:
```suggestion
当为 true 时，不会进行默认的位置合并，而是根据名称合并定义了 `name` 属性的对象数组元素，并且生成的数组将是两个输入数组的并集。
```

`instead of the default positional merging, ` is missed here.

Reply:
done